src/twisted/mail/pop3client.py

```diff
@@ -1,1238 +1,24 @@
-            a 2-tuple of (0) L{int}, (1) L{int} or fails with
-            L{ServerErrorResponse}
-        @return: A deferred which fires when the server response is received.
-            On an OK response, the deferred succeeds with the number of
-            messages in the mailbox and the size of the mailbox in octets.
-            On an ERR response, the deferred fails with a server error
-            response failure.
-        """
-        return self.sendShort(b"STAT", None).addCallback(_statXform)
-
-    def listSize(self, consumer=None):
-        """
-        Send a LIST command to retrieve the sizes of all messages on the
-        server.
-
-        @type consumer: L{None} or callable that takes
-            2-L{tuple} of (0) L{int}, (1) L{int}
-        @param consumer: A function which consumes the 0-based message index
-            and message size derived from the server response.
-
-        @rtype: L{Deferred <defer.Deferred>} which fires L{list} of L{int} or
-            callable that takes 2-L{tuple} of (0) L{int}, (1) L{int}
-        @return: A deferred which fires when the entire response has been
-            received.  When a consumer is not provided, the return value is a
-            list of message sizes.  Otherwise, it returns the consumer itself.
-        """
-        return self._consumeOrSetItem(b"LIST", None, consumer, _listXform)
-
-    def listUID(self, consumer=None):
-        """
-        Send a UIDL command to retrieve the UIDs of all messages on the server.
-
-        @type consumer: L{None} or callable that takes
-            2-L{tuple} of (0) L{int}, (1) L{bytes}
-        @param consumer: A function which consumes the 0-based message index
-            and UID derived from the server response.
-
-        @rtype: L{Deferred <defer.Deferred>} which fires with L{list} of
-            L{object} or callable that takes 2-L{tuple} of (0) L{int},
-            (1) L{bytes}
-        @return: A deferred which fires when the entire response has been
-            received.  When a consumer is not provided, the return value is a
-            list of message sizes.  Otherwise, it returns the consumer itself.
-        """
-        return self._consumeOrSetItem(b"UIDL", None, consumer, _uidXform)
-
-    def quit(self):
-        """
-        Send a QUIT command to disconnect from the server.
-
-        @rtype: L{Deferred <defer.Deferred>} which successfully fires with
-            L{bytes} or fails with L{ServerErrorResponse}
-        @return: A deferred which fires when the server response is received.
-            On an OK response, the deferred succeeds with the server
-            response minus the status indicator.  On an ERR response, the
-            deferred fails with a server error response failure.
-        """
-        return self.sendShort(b"QUIT", None)
-
+warnings.warn(
+    "twisted.mail.pop3client was deprecated in NEXT. " "Use twisted.mail.pop3 instead.",
```

Comment:
```suggestion
    "twisted.mail.pop3client was deprecated in NEXT. Use twisted.mail.pop3 instead.",
```

Reply:
In other deprecations, `Twisted NEXT` is used instead of just `NEXT`. I don't know if that's a requirement for the version substitution to work, but it's probably a good idea to be consistent in any case.

Reply:
Yes. My bad. It should be "Twisted NEXT" as NEXT is relaced with just the version not "product_name + version "

Reply:
The `NEXT. " "Use` part was done by black .

Reply:
@adiroiban @mthuurne you can catch this with https://github.com/keisheiled/flake8-implicit-str-concat

but `black --experimental-string-processing` should fix them soon